src/main/java/com/powsybl/openloadflow/network/impl/PropagatedContingency.java

```diff
@@ -164,7 +167,26 @@ private static PropagatedContingency create(Network network, Contingency conting
                     if (hvdcLine == null) {
                         throw new PowsyblException("HVDC line '" + element.getId() + "' not found in the network");
                     }
-                    hvdcIdsToOpen.add(element.getId());
+                    HvdcAngleDroopActivePowerControl control = hvdcLine.getExtension(HvdcAngleDroopActivePowerControl.class);
+                    if (control != null && control.isEnabled() && hvdcAcEmulation) {
+                        // Not depending on AC or DC computation.
+                        throw new PowsyblException("Contingency on HVDC line '" + element.getId() + "' with AC emulation control: not supported yet.");
```

Comment:
@Hadrien-Godard and @geofjamg I am about to support this feature indeed for AC sensitivity analysis and for AC security analysis, but in another PR...